xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVTB.cpp

```diff
@@ -160,6 +183,11 @@ bool CRendererVTB::RenderHook(int index)
   return true;
 }
 
+void CRendererVTB::AddSupportedHwRenderFormats()
```

Comment:
Yes, I am sure. At the time a decoder is crated, there is no instance of a renderer and m_format may be empty anyway. I will drop m_formats later.